plugin-template

```diff
@@ -332,6 +333,7 @@ def main():
 
     if "github" in sections:
         migrate_pytest_pin(plugin_root_dir)
```

Comment:
I guess we can safely stop attempting the pytest migration by now. WDYT?

Reply:
I see its been around for about 4 months, so probably.